context-linux/src/etc/one-context.d/net-99-report-ready

```diff
@@ -69,14 +69,29 @@ while [ "$RETRY_COUNT" -gt 0 ] ; do
             fi
             ;;
     esac
+}
+
+while [ "$RETRY_COUNT" -gt 0 ] ; do
+    if [[ -n "$READY_SCRIPT" ]];then
+        if eval "$READY_SCRIPT";then
+            report_ready
+        else
+            echo READY_SCRIPT "$READY_SCRIPT" failed to execute. Not reporting READY on attempt "$RETRY_COUNT" >&2
```

Comment:
So judging from this line the script is supposed to be an one-liner, right? :thinking: Why?